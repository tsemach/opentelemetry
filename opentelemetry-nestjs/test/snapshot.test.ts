describe('Snapshot Example Test', () => {
  it('test simple shanpshot', () => {
    expect({ a: 1, b: 2 }).toMatchSnapshot();
  });
});
