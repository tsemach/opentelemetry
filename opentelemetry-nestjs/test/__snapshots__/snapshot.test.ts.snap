// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Example Test test simple shanpshot 1`] = `
{
  "a": 1,
  "b": 2,
}
`;
